src/UITests/AIWinSession.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using Axe.Windows.Automation;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Appium.Windows;
+using OpenQA.Selenium.Remote;
+
+namespace UITests
+{
+    [TestClass]
+    public class AIWinSession
+    {
+        protected const string WindowsApplicationDriverUrl = "http://127.0.0.1:4723";
+
+        protected static WindowsDriver<WindowsElement> session;
+        protected static int processId;
+
+        public static void Setup(TestContext context)
+        {
+            // Launch a new instance of application
+            if (session == null)
```

Comment:
I'm sure you copied this code, but you could easily get rid of one layer of nesting by testing if `Session` was not null and returning, thus bringing the actual story of the function one level up.